project-card.js

```diff
@@ -0,0 +1,126 @@
+const project = [
+  {
+    Name: 'Profesional Art Printing Data',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    Featured_image: '/img/card-bg1.png',
+    Technologies: ['HTML', 'css', 'ruby'],
+    liveVersion: 'https://enning94.github.io/Personal-portfolio/',
+    sourceLink: 'https://github.com/Enning94/Personal-portfolio',
+  },
+  {
+    Name: 'Profesional Art Printing Data',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    Featured_image: '/img/card-bg1.png',
+    Technologies: ['HTML', 'css', 'ruby'],
+    liveVersion: 'https://enning94.github.io/Personal-portfolio/',
+    sourceLink: 'https://github.com/Enning94/Personal-portfolio',
+  },
+  {
+    Name: 'Profesional Art Printing Data',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    Featured_image: '/img/card-bg1.png',
+    Technologies: ['HTML', 'css', 'ruby'],
+    liveVersion: 'https://enning94.github.io/Personal-portfolio/',
+    sourceLink: 'https://github.com/Enning94/Personal-portfolio',
+  },
+  {
+    Name: 'Profesional Art Printing Data',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    Featured_image: '/img/card-bg1.png',
+    Technologies: ['HTML', 'css', 'ruby'],
+    liveVersion: 'https://enning94.github.io/Personal-portfolio/',
+    sourceLink: 'https://github.com/Enning94/Personal-portfolio',
+  },
+  {
+    Name: 'Profesional Art Printing Data',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    Featured_image: '/img/card-bg1.png',
+    Technologies: ['HTML', 'css', 'ruby'],
+    liveVersion: 'https://enning94.github.io/Personal-portfolio/',
+    sourceLink: 'https://github.com/Enning94/Personal-portfolio',
+  },
+  {
+    Name: 'Profesional Art Printing Data',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    Featured_image: '/img/card-bg1.png',
+    Technologies: ['HTML', 'css', 'ruby'],
+    liveVersion: 'https://enning94.github.io/Personal-portfolio/',
+    sourceLink: 'https://github.com/Enning94/Personal-portfolio',
+  },
+];
```

Comment:
- [ ] Your design is not similar to the Figma design kindly make it similar to the design.
- [ ] kindly fix the buttons as well.
### Your design
![figma](https://user-images.githubusercontent.com/81727988/180445145-0969f1e9-713c-4c14-8cf8-322444742867.png)